Makefile

```diff
@@ -1,7 +1,3 @@
-# Refers to https://github.com/tikv/pd/blob/master/Makefile
-
-default: build
-
```

Comment:
What I'm not sure about is, do we need to keep the link?

Reply:
Maybe. But I don't find any similarities this Makefile with pd's.

Reply:
@chunshao90 Do you remember why we added this link before? 🤣

Reply:
I guess there are some similarities between them: https://github.com/tikv/pd/blob/master/Makefile#L143

Reply:
They are quite different and if it's not clearly driven from another file, we should try to reduce the IP burden.

Reply:
> They are quite different and if it's not clearly derived from another file, we should try to reduce the IP burden.

Makes sense to me.